src/main/java/task/Task.java

```diff
@@ -0,0 +1,107 @@
+package task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a task that the user adds in the chatbot.
+ */
+abstract public class Task {
+
+    public enum TaskType {
+        TODO, DEADLINE, EVENT;
+    }
+
+    private boolean isDone;
+    private String taskName;
```

Comment:
I like how you kept the class variables private, following the coding standard. Good job!